openmdao/core/total_jac.py

```diff
@@ -1494,9 +1499,16 @@ def compute_totals_approx(self, initialize=False):
             if model._coloring_info['coloring'] is not None:
                 model._update_wrt_matches(model._coloring_info)
 
+        if show_progress:
+            setattr(model._approx_schemes, "out_stream", out_stream)
```

Comment:
You can set directly without setattr.  e.g., `model._approx_schemes[method] = out_stream`  That way you don't need to call del either. Just set it back to None at the end.